client/src/common/components/DropdownMenu.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import MenuIcon from 'assets/images/ellipsis-v-solid.svg';
+
+class DropdownMenu extends Component {
+  state = {
+    hideMenu: true
+  };
+
+  hideMenu = () => {
+    this.setState({ hideMenu: true });
+    document.removeEventListener('click', this.hideMenu);
+    document.removeEventListener('keydown', this.onPressEscape);
+  };
+
+  showMenu = () => {
+    this.setState({ hideMenu: false });
+    document.addEventListener('click', this.hideMenu);
+    document.addEventListener('keydown', this.onPressEscape);
+  };
+
+  onPressEscape = e => (e.keyCode === 27 ? this.hideMenu() : null);
+
+  toggleMenuVisibility = () => {
+    const { hideMenu } = this.state;
+    hideMenu ? this.showMenu() : this.hideMenu();
+  };
+
+  render() {
+    const { hideMenu } = this.state;
+    const { children, menuItems } = this.props;
+    const menuButton = children || (
+      <div className="dropdown__wrapper">
+        <img alt="Menu Icon" className="dropdown__menu-icon" src={MenuIcon} />
+      </div>
+    );
+    return (
+      <a className="dropdown" href="#!" onClick={this.toggleMenuVisibility}>
```

Comment:
shouldn't this be already changed to buttons?

Reply:
I do not know why these changes are not here... I added them again